corehq/motech/urls.py

```diff
@@ -105,6 +109,12 @@
         {'repeater_type': 'CaseExpressionRepeater'},
         name=EditCaseExpressionRepeaterView.urlname
     ),
+    url(
+        r'^forwarding/edit/ArcGISFormExpressionRepeater/(?P<repeater_id>\w+)/$',
+        EditArcGISFormExpressionRepeaterView.as_view(),
+        {'repeater_type': 'CaseExpressionRepeater'},
```

Comment:
Is this `CaseExpressionRepeater` correct? Should it not be `ArcGISFormExpressionRepeater`?

Reply:
You are completely correct, it should be the latter. Addressed in [1ce1676](https://github.com/dimagi/commcare-hq/pull/34929/commits/1ce1676b93681dd364aa789e6c15386df7e4d5ab).